src/main/resources/application.yml

```diff
@@ -23,6 +23,9 @@ application:
     headers:
       Content-Type: "application/json"
   maxStepRecursions: 10
+  dslParameters:
+    DOMAIN_URL: "https://example.com"
+    PORT: 8080
```

Comment:
Remove this section as a whole from `application.yml` to `constants.ini`. This way, everything related to Ruuter's core setup remains separated from client-specific constants. And I mean _constants,_ not URLs specifically as these can be used throughout the DSL.